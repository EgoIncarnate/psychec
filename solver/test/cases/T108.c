void f() {
    int* p;
    *p = 0;

    T1* p1;
    *p1 = 0;

    T2* p2;
    *p2 = 10;
}
