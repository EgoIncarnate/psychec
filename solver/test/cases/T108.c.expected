typedef  int T2 ;
typedef  int /*<<< orphan or scalar*/ T1 ;
